src/main/java/main/Parser.java

```diff
@@ -0,0 +1,98 @@
+package main;
+
+import Objects.Deadline;
+import Objects.Event;
+import Objects.Task;
+import Objects.Todo;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+public class Parser {
+    Storage storage;
+
+    public static void input(TaskList taskList){
+        Scanner sc = new Scanner(System.in);
+        boolean isChanged = false;
+        while (true) {
+            if(isChanged){
+                try {
+                    Storage.writeToFile(taskList);
+                    isChanged = false;
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
```

Comment:
Should we leave the `IOException` to be handled elsewhere, instead of throwing a separate exception?